dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -108,6 +108,25 @@ dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 {% endif %}
 
+{% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %}
+{% if not skip_xcvrd %}
+[program:ycable]
+{% if delay_xcvrd %}
+command=bash -c "sleep 30 && {% if API_VERSION == 3 and 'ycable' not in python2_daemons %}nice -n -20 python3 {% else %} nice -n -20 python2 {% endif %}/usr/local/bin/ycable"
+{% else %}
+command=nice -n -20 {% if API_VERSION == 3 and 'ycable' not in python2_daemons %}python3 {% else %} python2 {% endif %}/usr/local/bin/ycable
+{% endif %}
+priority=6
```

Comment:
you seem to be reusing the 'priority' level of xcvrd. is this intentional?

Reply:
I am thinking about it, all daemons have a priority from 1-10. Maybe need to have this as 7 ?